crypto/rand/rand_unix.c

```diff
@@ -61,32 +60,24 @@ int RAND_poll(void)
     int i, k;
     struct timespec ts;
     unsigned char v;
-
 #  ifdef OPENSSL_SYS_VOS_HPPA
     long duration;
     extern void s$sleep(long *_duration, short int *_code);
 #  else
-#   ifdef OPENSSL_SYS_VOS_IA32
     long long duration;
     extern void s$sleep2(long long *_duration, short int *_code);
-#   else
-#    error "Unsupported Platform."
-#   endif                       /* OPENSSL_SYS_VOS_IA32 */
-#  endif                        /* OPENSSL_SYS_VOS_HPPA */
+#  endif
 
     /*
      * Seed with the gid, pid, and uid, to ensure *some* variation between
      * different processes.
      */
-
     curr_gid = getgid();
     RAND_add(&curr_gid, sizeof curr_gid, 1);
```

Comment:
This should count for zero randomness not one byte.

The value never changes between calls to RAND_poll, so even if it counts as randomness the first time through (which it probably shouldn't) it definitely shouldn't count afterwards.

Apologies for not picking this earlier.